.config/waybar/config

```diff
@@ -131,6 +131,22 @@
         "format-disconnected": "⚠",
         "format-alt": "{ifname} {essid} ({signalStrength}%)"
     },
+    "bluetooth": {
+	"format": "",
+	"format-disabled": "",
+	"format-off": "",
+	"format-connected": " {num_connections}",
+	// "format-connected": " {device_alias}",
+	"tooltip-format": "{controller_address} {status}\n\n{num_connections} connected",
+	"tooltip-format-disabled": "{status}",
+	"tooltip-format-connected": "{controller_address}\n\n{num_connections} connected\n\n{device_enumerate}",
+	"tooltip-format-enumerate-connected": "{device_alias}\t{device_address}",
+	"format-connected-battery": " {device_alias} {device_battery_percentage}%",
+	"tooltip-format-enumerate-connected-battery": "{device_alias}\t{device_address}\t{device_battery_percentage}%",
+	// "format-device-preference": [ "device1", "device2" ], // preference list deciding the displayed device
+	"on-click": "bluetooth toggle; pkill -SIGRTMIN+8 waybar",
+	"on-click-right": "exec alacritty -e sh -c 'bluetoothctl'"
```

Comment:
I don't know if the `bluez` and `tlp` packages are installed as a dependencies, but probably not. We probably need the spec for this to work, to get `bluetoothctl` and `bluetooth` tools.

Reply:
See, this is where it starts to break down. I don't want to add `Requires` upon `Requires` to the general case.

Ideally _this bit of configuration_ could be made conditional on the presence of a bluetooth device/software stack? But how to achieve that?

Reply:
Unfortunately, I don't see a way to template this config to be templated with smart conditions like that. I don't mind additional Bluetooth dependencies.

Reply:
Ok. On most desktop systems that want to use openSUSEway this should not be a big deal - they most likely will already have `tlp` and `bluez` installed, and if they don't, adding them should not be a huge deal. Let me amend this PR to include them as Recommends